apps/molgenis-components/src/components/forms/InputRefBack.vue

```diff
@@ -152,18 +189,57 @@ export default {
         );
       }
       return [];
-    }
+    },
   },
   methods: {
     getPrimaryKey,
-    async reload () {
-      this.data = await this.client.fetchTableDataValues(this.tableName, { filter: this.graphqlFilter });
-    }
+    async reload() {
+      this.isLoading = true;
+      this.data = await this.client.fetchTableDataValues(this.tableName, {
+        filter: this.graphqlFilter,
+      });
+      this.isLoading = false;
+    },
+    handleRowAction(type, key) {
+      this.editMode = type;
+      this.editRowPrimaryKey = key;
+      this.isEditModalShown = true;
+    },
+    handleModalClose() {
+      this.isEditModalShown = false;
+      this.reload();
+    },
+    handleDeleteRowRequest(key) {
+      this.editRowPrimaryKey = key;
+      this.isDeleteModalShown = true;
+    },
+    async handleExecuteDelete() {
+      this.isDeleteModalShown = false;
+      const resp = await this.client
+        .deleteRow(this.editRowPrimaryKey, this.tableName)
+        .catch(this.handleError);
+      if (resp) {
+        this.handleSuccess(resp);
+        this.reload();
+      }
+    },
+    handleError(error) {
+      if (Array.isArray(error?.response?.data?.errors)) {
+        this.graphqlError = error.response.data.errors[0].message;
+      } else {
+        this.graphqlError = error;
+      }
+      this.loading = false;
+    },
+    handleSuccess(resp) {
+      this.success = resp.data?.data?.delete?.message;
+    },
```

Comment:
success is never used, this code is dead

Reply:
true , setup for succes info banner , but will remove